app/app.go

```diff
@@ -127,6 +127,33 @@ func NewApp(c appConfig, db *sql.DB) (*App, error) {
 		}
 	}
 
+	if c.TLSListenAddr != "" {
+		var cert tls.Certificate
+		if c.TLSCertFile != "" && c.TLSKeyFile != "" {
```

Comment:
Can we put this stuff in a separate file & method?

```go
func (cfg appConfig) tlsConfig() (*tls.Config, error) {
 // do  `*tls.Config` stuff here
}
```

That will also make it easier to add ACME stuff later.